app/models/disability_compensation_submission.rb

```diff
@@ -1,7 +1,15 @@
 # frozen_string_literal: true
 
 class DisabilityCompensationSubmission < ActiveRecord::Base
+  enum status: {
+    submitted: 'submitted',
+    received: 'received',
+    retrying: 'retrying',
+    non_retryable_error: 'non_retryable_error',
+    exhausted: 'exhausted'
+  }
+
   validates :user_uuid, presence: true, uniqueness: { scope: :form_type }
   validates :form_type, presence: true, uniqueness: { scope: :user_uuid }
-  validates :claim_id, presence: true, uniqueness: true
+  validates :status, presence: true
```

Comment:
do we need to validate the presence of `job_id` and `response` as well?

Reply:
Good catch, yes for `job_id`, response can be null when the status is `submitted`